scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -131,6 +131,8 @@ proposals:
         host: vcs.qa.suse.de
         user: "@@vcenter_user@@"
         password: "@@vcenter_password@@"
+        cluster_name: []
```

Comment:
There is `ca_file:` attribute missing, the barclamp fails without providing this.